module/enrichers.mjs

```diff
@@ -78,6 +82,43 @@ function parseConfig(match) {
   return config;
 }
 
+/* -------------------------------------------- */
+/*  Attack Enricher                             */
+/* -------------------------------------------- */
+
+/**
+ * Enrich an attack link using a pre-set to hit value.
+ * @param {object} config              Configuration data.
+ * @param {string} [label]             Optional label to replace default text.
+ * @param {EnrichmentOptions} options  Options provided to customize text enrichment.
+ * @returns {HTMLElement|null}         An HTML link if the save could be built, otherwise null.}
+ *
+ * @example Create an attack link using a fixed to hit:
+ * ```[[/attack +5]]``` or ```[[/attack formula=5]]```
+ * becomes
+ * ```html
+ * <a class="roll-action" data-roll-action="attack" data-formula="+5">
+ *   <i class="fa-solid fa-dice-d20" inert></i> +5
+ * </a>
+ * ```
+ */
+async function enrichAttack(config, label, options) {
+  const formulaParts = [];
+  if ( config.formula ) formulaParts.push(config.formula);
+  for ( const value of config.values ) formulaParts.push(value);
+  config.formula = Roll.defaultImplementation.replaceFormulaData(formulaParts.join(" "), options.rollData ?? {});
+  if ( !config.formula.startsWith("+") && !config.formula.startsWith("-") ) config.formula = `+${config.formula}`;
+  config.type = "attack";
+
+  if ( label ) return createRollLink(label, config);
+
+  const span = document.createElement("span");
+  span.innerHTML = game.i18n.format(`EDITOR.DND5E.Inline.Attack${config.format === "long" ? "Long" : "Short"}`, {
+    formula: createRollLink(config.formula, config).outerHTML
```

Comment:
We should run the label through `simplifyBonus`. Without it, a simple formula like `[[/attack @prof + @abilities.str.mod]]` displays as "+2 + 2 to hit".

```suggestion
    formula: createRollLink(simplifyBonus(config.formula), config).outerHTML
```

Reply:
Probably `simplifyRollFormula`, cause we don't want to squash dice values, but yeah.